### PR TITLE
ENT-7266: Avoid creating WORKDIR/modules/$(_cmdb_net_request_state_file)

### PR DESCRIPTION
**Pull request by nickanderson**

### CODE REVIEW

**Comment by nickanderson** [comment]
@cf-bottom jenkins, please.

**Comment by cf-bottom** [comment]
Sure, I triggered a build:

[Build Status](https://ci.cfengine.com//job/pr-pipeline/6784/)

**Jenkins:** https://ci.cfengine.com/job/pr-pipeline/6784/

**Packages:** http://buildcache.cfengine.com/packages/testing-pr/jenkins-pr-pipeline-6784/

File: cfe_internal/update/update_policy.cf [hunk 1]
```diff
@@ -367,8 +368,8 @@ bundle agent cfe_internal_update_cmdb_data_distribution
       "$(_cmdb_next_request_state_file)"
         content => "0$(const.n)",
         handle => "cmdb_data_change_next_seed",
-        unless => fileexists("$(_cmdb_next_request_state_file)" ),
-        if => isvariable( "_cmdb_next_request_state_file" );
+        if => and( not(fileexists("$(_cmdb_next_request_state_file)" )),
```
Comment (craigcomstock) [inline cfe_internal/update/update_policy.cf:371]: why the change here? isn't it the same effect of logic either way?

It does seem a bit more clear and explicit the way you have changed things.

Comment (nickanderson) [reply cfe_internal/update/update_policy.cf:371]: \_(ツ)_/¯
